Update documentation website

ardislu opened the pull request:
:^)

ardislu committed:
Update favicon

ardislu committed:
Use Bootstrap for documentation site

Replace w3.css with Bootstrap and improve general aesthetics.

jzaturensky reviewed (changes requested):
Please fix broken side bar

ardislu committed:
Add quick links

New input fields to quickly create links for specific GET requests, with copy to clipboard buttons.

ardislu commented:
Side bar is fixed
